services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/PaymentMethodsTest.java

```diff
@@ -0,0 +1,86 @@
+package com.mapbox.api.directions.v5.models;
+
+import com.google.gson.JsonSyntaxException;
+import com.mapbox.core.TestUtils;
+import org.junit.Test;
+import org.junit.function.ThrowingRunnable;
+
+import static com.mapbox.core.TestUtils.deserialize;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertThrows;
+
+public class PaymentMethodsTest {
+
+    @Test
+    public void sanity() throws Exception {
+        PaymentMethods paymentMethods = PaymentMethods.builder().etc(PaymentMethod.builder().small(555.6).build()).build();
+        assertNotNull(paymentMethods);
+    }
+
+    @Test
+    public void serializable() throws Exception {
+        PaymentMethods paymentMethods = PaymentMethods.builder()
+                .etc(PaymentMethod.builder().small(555.6).build())
+                .cash(PaymentMethod.builder().standard(777.0).build())
+                .build();
+        byte[] serialized = TestUtils.serialize(paymentMethods);
+        assertEquals(paymentMethods, deserialize(serialized, PaymentMethods.class));
+    }
+
+    @Test
+    public void toFromJsonFilled() {
+        PaymentMethod etc = PaymentMethod.builder().small(555.6).build();
+        PaymentMethod cash = PaymentMethod.builder().standard(777.0).build();
+        PaymentMethods original = PaymentMethods.builder()
+                .etc(etc)
+                .cash(cash)
+                .build();
+        String json = original.toJson();
+        PaymentMethods deserialized = PaymentMethods.fromJson(json);
+        assertEquals(etc, deserialized.etc());
+        assertEquals(cash, deserialized.cash());
```

Comment:
why do you compare separate fields? Isn't it easier to compare 2 objects?
```suggestion
        PaymentMethod etc = PaymentMethod.builder().small(555.6).build();
        PaymentMethod cash = PaymentMethod.builder().standard(777.0).build();
        PaymentMethods original = PaymentMethods.builder()
                .etc(etc)
                .cash(cash)
                .build();
        String json = original.toJson();
        PaymentMethods deserialized = PaymentMethods.fromJson(json);
        assertEquals(original, deserialized);
```